plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -81,14 +90,24 @@ export async function createRouter(
       });
     })
     .post('/v1/jobs', async (req, res) => {
-      const template: TemplateEntityV1alpha1 = req.body.template;
+      const templateName: string = req.body.templateName;
```

Comment:
just want to double check - is this on the full ref form `<kind>:<namespace>/<name>`?

Reply:
No but it should be, maybe we can raise a separate issue for this though as it only queries in the default namespace for templates anyway.

Reply:
Is it same issue which @Rugvip mentioned below?

Reply:
In a sense, yes. We can leave it as just the metadata.name for now since that's how the logic actually works, and revisit at a later time